controller_manager:
  ros__parameters:

    update_rate: 1000

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    manipulator_controller:
      type: joint_trajectory_controller/JointTrajectoryController

    forward_position_controller:
      type: position_controllers/JointGroupPositionController


manipulator_controller:
  ros__parameters:
    joints:
      - joint_1
      - joint_2
      - joint_3
      - joint_4
      - joint_5
      - joint_6
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
    open_loop_control: true
    constraints:
      goal_time: 0.0


forward_position_controller:
  ros__parameters:
    joints:
      - joint_1
      - joint_2
      - joint_3
      - joint_4
      - joint_5
      - joint_6